### ISSUE TITLE
The "bookmark" button cannot be operated by assistive devices.

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3034]
Visually hide the <input> element rather than setting it to display:none so the operable tab stop remains.

https://www.w3.org/WAI/WCAG21/quickref/#name-role-value
